CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Add the ability to pass a list of HTTPoison options to `Rest.post` and `Rest.put`
```

Comment:
Can you please add a secondary comment specifying that this is also available via the new optional 4th param to `ShopifyAPI.REST.Fulfillment.create`.